### DISCUSSION TITLE
Doris FE Mac 本地开发环境搭建 - IntelliJ IDEA完运行内存不够，这个最低需要多少内存啊🤔

### DISCUSSION THREAD

**Discussion by chenghui-ch** [apache/doris#32728] [RFC]
编译的branch-2.1。
按照这个文档配置的：[Doris FE Mac 开发环境搭建 - IntelliJ IDEA](https://doris.apache.org/zh-CN/community/developer-guide/mac-dev/fe-idea-dev)

<img width="865" alt="运行情况" src="https://github.com/apache/doris/assets/48585272/2b380be7-8284-499d-b43b-918466382a29">

<img width="490" alt="配置信息" src="https://github.com/apache/doris/assets/48585272/fdfd3a32-e01d-4516-97f4-4913a7cfa046">

**Comment by ihadoop** [comment]
你截图的那个配置的是运行内存，报错的是编译内存。试下修改这个配置，如下图
![image](https://github.com/apache/doris/assets/5036560/65fc9ddc-f04b-4ad9-b574-c7e7830ac80a)

**Reply by chenghui-ch** [reply]
多谢。可以了😌

**Comment by Hanfee** [comment]
Thank you, followed your instructions and solved the problem perfectly
